AnnoyancesFilter/sections/popups.txt

```diff
@@ -118,6 +118,7 @@ revouninstaller.com#$##popupModal { display: none !important; }
 revouninstaller.com#$#.modal-backdrop { display: none !important; }
 revouninstaller.com#$#body { overflow: visible !important; padding-right: 0 !important; }
 techmeme.com###nlsb
+kompas.com##.root > div.dark-fixed
```

Comment:
I'm not sure that we should hide this element, this is part of site's functionality

Reply:
I thought so too, but it covers the content and is really annoying.

Reply:
@Alex-302 decide =)